File: branches/Andean/Fantasy.Base/ServiceModel/CallbackExpiredException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;

namespace Fantasy.ServiceModel
{
   
    [DataContract]
    public class CallbackExpiredException
    {
        public CallbackExpiredException() 
        {
            this.Message = "Callback expired.";
        }

        [DataMember]
        public string Message { get; set; }
    }
}

File: branches/Stheno/Fantasy.Base/src/fantasy/servicemodel/CallbackExpiredException.java
package fantasy.servicemodel;

public class CallbackExpiredException extends RuntimeException
{
	/**
	 * 
	 */
	private static final long serialVersionUID = -6215782633571433993L;
	public CallbackExpiredException()
	{
		this.setMessage("Callback expired.");
	}

	private String privateMessage;
	public final String getMessage()
	{
		return privateMessage;
	}
	public final void setMessage(String value)
	{
		privateMessage = value;
	}
}